File: micronaut-java-parent-example/domain/src/test/java/io/archimedesfw/maven/micronaut/example/todo/TodoTest.java
package io.archimedesfw.maven.micronaut.example.todo;

import org.junit.jupiter.api.Test;

import java.time.LocalDateTime;

import static org.junit.jupiter.api.Assertions.assertEquals;

class TodoTest {

    private static final LocalDateTime TS = LocalDateTime.now();

    @Test
    void equals() {
        final var todo1 = new Todo(1, "title", TS, null);
        final var todo2 = new Todo(1, "title", TS, null);

        assertEquals(todo1, todo2);
    }

}

File: micronaut-kotlin-parent-example/domain/src/test/kotlin/io/archimedesfw/maven/micronaut/example/todo/TodoTest.kt
package io.archimedesfw.maven.micronaut.example.todo

import org.junit.jupiter.api.Test

import java.time.LocalDateTime

import org.junit.jupiter.api.Assertions.assertEquals

internal class TodoTest {

    @Test
    fun equals() {
        val todo1 = Todo(1, "title", TS, null)
        val todo2 = Todo(1, "title", TS, null)

        assertEquals(todo1, todo2)
    }

    companion object {

        private val TS = LocalDateTime.now()
    }

}
